src/Components/ProgramListPage.js

```diff
@@ -1,23 +1,94 @@
-import React from 'react';
-import { StyleSheet, View, FlatList } from 'react-native';
+import React, { useState } from 'react';
+import { ScrollView, StyleSheet, Text, View, TouchableOpacity, FlatList, Image } from 'react-native';
 import { connect } from 'react-redux';
+import * as Animatable from 'react-native-animatable';
+import Accordion from 'react-native-collapsible/Accordion';
+import { MaterialCommunityIcons } from '@expo/vector-icons';
 
 import I18n from '../utils/i18n';
 import theme from '../styles/theme.style';
 import Program from './programs/Program';
 import Button from './shared/Button';
+import arrowDark from '../../assets/arrow_dark.png';
 
 export const ProgramListPage = props => {
-  const { programs, navigation } = props;
+  const { navigation, programs } = props;
+
+  const [activeSections, setActiveSections] = useState([undefined]);
+
+  const setSections = sections => {
+    setActiveSections(sections.includes(undefined) ? [] : sections);
```

Comment:
Does the `sections` variable equals `[undefined]` at this time or is it just `[]`?

If it's `[]` we could directly call `    setActiveSections(sections);`